ui/framework/src/test/imodel-components/VisibilityTreeEventHandler.test.ts

```diff
@@ -0,0 +1,119 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { expect } from "chai";
+import * as sinon from "sinon";
+import { BeEvent, BeUiEvent, using } from "@bentley/bentleyjs-core";
+import * as moq from "@bentley/presentation-common/lib/test/_helpers/Mocks";
+import { IVisibilityHandler, VisibilityChangeListener, VisibilityStatus, VisibilityTreeEventHandler, VisibilityTreeEventHandlerParams } from "../../ui-framework/imodel-components/VisibilityTreeEventHandler";
+import { AbstractTreeNodeLoaderWithProvider, TreeModel, TreeModelChanges, TreeModelNode, TreeModelSource } from "@bentley/ui-components";
+import { SelectionHandler } from "@bentley/presentation-frontend";
+import { IPresentationTreeDataProvider } from "@bentley/presentation-components";
+import { createSimpleTreeModelNode } from "./Common";
+import TestUtils from "../TestUtils";
+
+describe("VisibilityTreeEventHandler", () => {
+
+  const modelSourceMock = moq.Mock.ofType<TreeModelSource>();
+  const modelMock = moq.Mock.ofType<TreeModel>();
+  const nodeLoaderMock = moq.Mock.ofType<AbstractTreeNodeLoaderWithProvider<IPresentationTreeDataProvider>>();
+  const dataProviderMock = moq.Mock.ofType<IPresentationTreeDataProvider>();
+  const selectionHandlerMock = moq.Mock.ofType<SelectionHandler>();
+
+  const changeVisibility = sinon.fake();
+  const getVisibilityStatus = sinon.stub();
+  const dispose = sinon.fake();
+  const onVisibilityChange = new BeEvent<VisibilityChangeListener>();
+
+  const visibilityHandler: IVisibilityHandler = {
+    changeVisibility,
+    getVisibilityStatus,
+    onVisibilityChange,
+    dispose,
+  };
```

Comment:
```suggestion
  const visibilityHandler: IVisibilityHandler = {
    changeVisibility: sinon.fake(),
    getVisibilityStatus,
    onVisibilityChange,
    dispose: sinon.fake(),
  };
```

And you can remove `dispose` and `changeVisibility` local variables as they are not used in tests anywhere.

Reply:
Thanks, updated it